astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,325 @@
+```
+
+## Conclusion
+
+Your directory for the `complete-application` tree should look like this:
+
+```text
+└── complete-application
+    ├── appsettings.Development.json
+    ├── appsettings.json
+    ├── bin/
+        ├── ...
+    ├── obj/
+        ├── ...
+    ├── Pages/
+    │   ├── _ViewImports.cshtml
+    │   ├── _ViewStart.cshtml
+    |   |── Account.cshtml
+    |   |── Accuont.cshtml.cs
+    │   ├── Error.cshtml
+    │   ├── Error.cshtml.cs
+    │   ├── Index.cshtml
+    │   ├── Index.cshtml.cs
+    │   ├── Logout.cshtml
+    │   ├── Logout.cshtml.cs
+    |   |── MakeChange.cshtml
+    |   |── MakeChange.cshtml.cs
+    │   ├── Privacy.cshtml
+    │   ├── Privacy.cshtml.cs
+    │   └── Shared/
+    │       ├── _Layout.cshtml
+    │       ├── _Layout.cshtml.css
+    │       └── _ValidateScriptsPartial.cshtml
+    ├── Program.cs
+    ├── Properties/
+        ├── ...
+    ├── Startup.cs
+    └── wwwroot/
+        ├── ...
+```
+
+Once you've created this directory structure, you can start up the {frontmatter.technology} application using this command:
+
+```shell
+cd complete-application
+dotnet publish -r osx-x64
+dotnet run
+```
+
+The full code is available [here](https://github.com/FusionAuth/fusionauth-quickstart-dotnet-web).
+
+## Troubleshooting
+
+If you run into an issue with cookies on Safari or other browsers, you might need to run the ASP.NET application under SSL.
+
+You can follow [this guide](https://learn.microsoft.com/en-us/dotnet/core/additional-tools/self-signed-certificates-guide) to install development SSL certificates for your .NET environment.
+
+Alternatively, you can run the project using [Visual Studio](https://visualstudio.microsoft.com), which will run the project using SSL.
+
+If you do this, make sure to update the `Authorized Redirect URL` to reflect the `https` protocol. Also note that the project will probably run on a different port when using SSL, so you must update that as well. To do so, log into the administrative user interface, navigate to `Applications`, then click the `Edit` button on your application and navigate to the `OAuth` tab. You can have more than one URL.
+
+## Next Steps
+This quickstart is a great way to get a proof of concept up and running quickly, but to run
+your application in production, there are some things you're going to want to do.
+
+### FusionAuth Customization
+FusionAuth gives you the ability to customize just about everything with the user's experience and your application's integration. This includes
+* [Hosted pages](/docs/v1/tech/themes/) such as login, registration, email verification, and many more
+* [Email templates](/docs/v1/tech/email-templates/email-templates)
+* [User data and custom claims in access token JWTs](/articles/tokens/jwt-components-explained)
+
+### Security
+* [OmniAuth OpenIDConnect](https://github.com/omniauth/omniauth) handles token validation and refresh
```

Comment:
We don't want to point to ruby doc here.

Reply:
No we don't.  Fixed.